workshops/image_editor/README.md

```diff
@@ -0,0 +1,451 @@
+}
+```
+Let's break this down:
+In this code we are creating a for statement, in which we create a variable called `i` that starts at 0, on each iteration, we check if the condition is false or true, or in other words, whether `i` is smaller than the amount of elements in `inputrange`.
+
+- If the condition is true, for each input it adds an event listener to it, and when the event happens, the `editimage` function gets called.
+- If the condition is false, we stop the loop.
+
+ In order to get to the next iteration, we add one to `i` using `i++`.
+
+Now we'll create a function that takes the values from the range inputs, and then update the style of the image applying the css filter function using those values:
+
+```js
+function editimage() {
+  let gs = document.getElementById('gs');
+  let blur = document.getElementById('blur');
+  let huerotate = document.getElementById('hue-rotate');
+  let sepia = document.getElementById('sepia');
+
+  let gsval = gs.value;
+  let blurval = blur.value;
+  let huerotateval = huerotate.value;
+  let sepiaval = sepia.value;
+
+  targetimage.style.filter = 'grayscale('+gsval+'%) blur('+blurval+'px) hue-rotate('+huerotateval+'deg) sepia('+sepiaval+'%) invert('+invertval+'%)';
+}
+```
+Breaking down:
+1. The first 4 variables hold the range inputs from the HTML document using the method: ```document.getElementById('')``` 
+2. The another 4 variables take those range inputs and get them actual values, using the `value` property
+3. Then we take the `targetimage` element and we style it using the `style` & `filter` properties, and we specify what filters we want to apply.
+
+Let's test it:
+![Image displaying gif](https://cloud-rnyr55z84.vercel.app/0screen_recording_2020-11-08_at_11.31.24_pm.gif)
+
+Finally we'll need to give the reset button its functionality:
+```js
+let sliderform = document.getElementById('slider-form');
+sliderform.addEventListener('reset', function() {
+  sliderform.reset();
+  setTimeout(function() {
+    editimage();
+  }, 0)
+})
+```
+Let's break it down:
+- We are creating a variable that holds the form in section 3.
+- Then we are adding it an event listener, that when the button is clicked a function is called. What this function does is to reset the values for all the inputs inside of the form using the `reset()` method, and then using the `setTimeout()` method we call the function `editimage` in the given time (0 seconds).
+
+Let's try it:
+![Image displaying gif](https://cloud-6xk3j2yw1.vercel.app/0screen_recording_2020-11-08_at_11.35.41_pm.gif)
+
+## Hack it
+Congratulations! You just made an amazing image editor. 
+![Congrats gif](https://cloud-d0aqa4icc.vercel.app/0bec38a05d56ac6ae2d9dec2f482ebff9.gif)
+Hack this workshop and share it in the [Hack Club Slack](https://hackclub.slack.com/), I would love to see what you can create using what you've learned in this workshop!
+
+## Other examples
+- [Example 1](https://webdevtrick.com/demos/css-filter-editor/)
```

Comment:
You should be a little more descriptive here than just "example 1" and "example 2". What do these examples do?